tests/tests_mqtt_server/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "tests_mqtt_server"
```

```suggestion
name = "test_mqtt_server"
```
Wondering why it was plural?